drivers/hmis/app/graphql/types/hmis_schema/ce_opportunity.rb

```diff
@@ -0,0 +1,29 @@
+###
+# Copyright 2016 - 2024 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+# frozen_string_literal: true
+
+module Types
+  class HmisSchema::CeOpportunity < Types::BaseObject
+    field :id, ID, null: false
+    field :name, String, null: false
```

Comment:
I'm guessing these won't have names themselves, or at least not one that is often set by the person creating the opportunity.